huntsman/utils/pyro/__init__.py

```diff
@@ -1,77 +1,13 @@
 from warnings import warn
 
-import netifaces
 import Pyro4
 from Pyro4 import naming, errors
 
-from huntsman.utils import load_config, sshfs_mount
+from huntsman.utils import get_own_ip, load_config, sshfs_mount
+from huntsman.utils.config import query_config_server
 from huntsman.camera.pyro import CameraServer
-from panoptes.utils.config import client
-
-
-def get_own_ip(verbose=False, logger=None):
```

Comment:
If you haven't already, can you make sure that . any other use of `get_own_ip` now points at the new location?